[4.4] windows export template failing to build from linux mono editor

### Tested versions

encountered with 4.4 beta (commit 9ee1873)

### System information

godot 4.4 on ubuntu based docker image running through woodpecker CI on an 11th gen intel nuc

### Issue description

got an error while compiling/building the windows export templates from a linux editor (.net build), here is the last several lines including the error:
```
Compiling thirdparty/volk/volk.c ...
Compiling thirdparty/vulkan/vk_mem_alloc.cpp ...
Compiling thirdparty/glad/gl.c ...
Compiling thirdparty/glad/egl.c ...
Compiling drivers/gles3/rasterizer_canvas_gles3.cpp ...
In file included from /usr/share/mingw-w64/include/corecrt.h:10,
                 from /usr/share/mingw-w64/include/crtdefs.h:10,
                 from /usr/share/mingw-w64/include/stddef.h:7,
                 from /usr/lib/gcc/x86_64-w64-mingw32/10-posix/include/stddef.h:1,
                 from ./core/typedefs.h:34,
                 from ./core/os/mutex.h:34,
                 from drivers/wasapi/audio_driver_wasapi.h:36,
                 from drivers/wasapi/audio_driver_wasapi.cpp:33:
drivers/wasapi/audio_driver_wasapi.cpp:89:1: error: expected unqualified-id before string constant
   89 | __CRT_UUID_DECL(IAudioClient3, 0x7ED4EE07, 0x8E67, 0x4CD4, 0x8C, 0x1A, 0x2B, 0x7A, 0x59, 0x87, 0xAD, 0x42)
      | ^~~~~~~~~~~~~~~
Compiling drivers/gles3/rasterizer_gles3.cpp ...
scons: *** [drivers/wasapi/audio_driver_wasapi.windows.template_debug.x86_64.o] Error 1
scons: building terminated because of errors.
INFO: Time elapsed: 00:06:48.85 
```



### Steps to reproduce

here is the pipeline i used, for those unfamilliar with woodpecker, everything in the /woodpecker directory gets shared between steps


[GodotBuild.txt](https://github.com/user-attachments/files/18613495/GodotBuild.txt)
https://gist.github.com/hpf3/32300c0cebb64b9c995aca3ba54b225a

[Dockerfile.txt](https://github.com/user-attachments/files/18613448/Dockerfile.txt)
https://gist.github.com/hpf3/515163c9c78c8e83351a48ad0f143b8f

### Minimal reproduction project (MRP)

n/a

--- a/drivers/wasapi/audio_driver_wasapi.cpp
+++ b/drivers/wasapi/audio_driver_wasapi.cpp
@@ -55,6 +55,7 @@ typedef interface IAudioClient3 IAudioClient3;
 #ifndef __IAudioClient3_INTERFACE_DEFINED__
 #define __IAudioClient3_INTERFACE_DEFINED__
 
+// clang-format off
 MIDL_INTERFACE("7ED4EE07-8E67-4CD4-8C1A-2B7A5987AD42")
 IAudioClient3 : public IAudioClient2 {
 public:
@@ -85,7 +86,8 @@ public:
 			_In_ const WAVEFORMATEX *pFormat,
 			/* [annotation][in] */
 			_In_opt_ LPCGUID AudioSessionGuid) = 0;
-}
+};
+// clang-format on
 __CRT_UUID_DECL(IAudioClient3, 0x7ED4EE07, 0x8E67, 0x4CD4, 0x8C, 0x1A, 0x2B, 0x7A, 0x59, 0x87, 0xAD, 0x42)
 
 #endif // __IAudioClient3_INTERFACE_DEFINED__

